ml-application-framework/src/components/RecordRaw/RecordRaw.stories.ts

```diff
@@ -0,0 +1,115 @@
+import type { Meta, StoryObj } from '@storybook/react'
```

Comment:
Can we include actual descriptions of the allowed properties in the description fields? Similar to this:

https://mayflower.digital.mass.gov/react/index.html?path=/story/atoms-buttons-button--button-example

For this widget's descriptions, we can use something similar to what's here:

https://github.com/mac-s-g/react-json-view

Suggested descriptions:

data - The JSON data to display. [string]
rootName - Name of the root node. [string]
maxHeight - Maximum height of the container. Content that exceeds this height will scroll. [string]
enableClipboard - Show clickable icons for copying object and array data. [boolean]
displayDataTypes - Show data types as prefixes to values. [boolean]
quotesOnKeys - Include quotes around keys. [boolean]
displayObjectSize - Show size of objects and arrays. [boolean]
indentWidth - Indent width for nested objects. [number]
groupArraysAfterLength - Display array values in groups based on the value. Groups are displayed with bracket notation and can be expanded and collapsed by clicking the brackets. [number]